### DISCUSSION TITLE
既然要做开源，demo也要做到跨平台呀，这方面有点减分了(我是Mac)

### DISCUSSION THREAD

**Discussion by nwuking** [alibaba/xquic#275] [Ideas]
1、sendmmsg在Mac不支持(不知道是不是我电脑特殊)

2、./demo_server —— 报错退出了，xqc_engine_create error

3、加油！

**Comment by ghost** [comment]
跑test_server

**Comment by ruiqizhou** [comment]
https://github.com/alibaba/xquic/blob/main/docs/docs-zh/Testing-zh.md
